tools/test.py

```diff
@@ -255,6 +255,10 @@ def HasRun(self, output):
         logger.info('#' + l)
       for l in output.output.stdout.splitlines():
         logger.info('#' + l)
+    elif output.HasSkipped():
+      skip = re.findall('# SKIP\S+ (.+)', output.output.stdout,
```

Comment:
It is better to use raw strings for RegEx. Also, we can precompile and store the RegEx with `re.compile`